src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -120,11 +121,10 @@ import LegendLayerInfos = __esri.LegendLayerInfos;
 import ScreenPoint = __esri.ScreenPoint;
 import RasterStretchRenderer from "@arcgis/core/renderers/RasterStretchRenderer";
 import DimensionalDefinition from "@arcgis/core/layers/support/DimensionalDefinition";
```

Comment:
are the color ramps used in Pro? why were they removed/added in the first place?

Reply:
I think they were moved to `jsBuilder.ts`, and didn't get cleaned up here, on a previous PR.